frontend/src/app/[locale]/formPrototype/[applicationId]/page.tsx

```diff
@@ -0,0 +1,93 @@
+import { Metadata } from "next";
+import NotFound from "src/app/[locale]/not-found";
+import { COMPETITION_ID } from "src/constants/competitions";
+import { ApiRequestError, parseErrorStatus } from "src/errors";
+import withFeatureFlag from "src/hoc/withFeatureFlag";
+import { getCompetitionDetails } from "src/services/fetch/fetchers/competitionsFetcher";
+import { FormDetail } from "src/types/formResponseTypes";
+
+import Link from "next/link";
+import { redirect } from "next/navigation";
+import { GridContainer } from "@trussworks/react-uswds";
+
+import BetaAlert from "src/components/BetaAlert";
+
+export const dynamic = "force-dynamic";
+
+export function generateMetadata() {
+  const meta: Metadata = {
+    title: `Form demo application landing page`,
+  };
+  return meta;
+}
+
+interface ApplicactionLandingPageProps {
+  params: Promise<{ applicationId: string; locale: string }>;
+}
+
+const FormLinks = ({
+  forms,
+  applicationId,
+}: {
+  forms: [{ form: FormDetail }];
+  applicationId: string;
+}) => {
+  if (forms.length > 0) {
+    return (
+      <ul>
+        {forms.map((form) => {
+          return (
+            <li key={form.form.form_name}>
+              <Link
+                href={`/formPrototype/${applicationId}/${form.form.form_id}`}
+              >
+                {form.form.form_name}
+              </Link>
+            </li>
+          );
+        })}
+      </ul>
+    );
+  } else {
+    return <></>;
+  }
+};
+
+async function ApplicactionLandingPage({
+  params,
+}: ApplicactionLandingPageProps) {
+  const { applicationId } = await params;
+  let forms = [] as unknown as [{ form: FormDetail }];
```

Comment:
this typing is awkward and I don't think it's necessary - works fine locally for me with
```suggestion
  let forms = [];
```
but that could be my local acting funny